ckanext/scheming/templates/scheming/snippets/form_field.html

```diff
@@ -58,6 +60,25 @@
 
     <div name="repeating-template" style="display:none">{{ repeating_panel('REPEATING-INDEX0', 'REPEATING-INDEX1') }}</div>
   </fieldset>
+
+{% block removal_popup %}
+  <div class="modal" id="mid-screen" tabindex="-1" role="dialog" aria-labelledby="mid-screenLabel" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="mid-screenLabel">{{ _('Confirmation of Removal') }}</h5>
+        </div>
+        <div class="modal-body" style="text-align:center;">
+            <p id="statusText0">{{ _('Are you sure you want to remove') }} {{ h.scheming_language_text(field.label) }} </p><p id="statusText1"></p>
```

Comment:
It's better to include the full sentence with a placeholder so that it can be properly formatted in different languages. Can you try this?
```suggestion
            <p id="statusText0">{{ _('Are you sure you want to remove {0}?').format(h.scheming_language_text(field.label) }} </p><p id="statusText1"></p>
```

Reply:
@wardi this code is outdated, does not apply in the latest commit